### ISSUE TITLE
Packaging tests on 5.6 time out bringing up vagrant boxes

### ISSUE THREAD

**Issue by cbuescher** [elastic/elasticsearch#36040]
Seen something this twice so far today: https://internal-ci.elastic.co/job/elastic+x-pack-elasticsearch+5.6+packaging-tests/235/console

```
11:36:02 ==> oel-7: Importing base box 'elastic/oraclelinux-7-x86_64'...
11:36:03 
11:36:07 Progress: 10%
11:36:08 Progress: 20%
11:36:10 Progress: 30%
11:36:13 Progress: 50%
11:36:13 Progress: 60%
11:36:17 Progress: 70%
11:36:19 Progress: 80%
11:36:31 Progress: 90%
11:36:31 ==> oel-7: Matching MAC address for NAT networking...
11:36:31 ==> oel-7: Checking if box 'elastic/oraclelinux-7-x86_64' is up to date...
11:36:32 ==> oel-7: Setting the name of the VM: elasticsearch_oel-7_1543487792326_84044
11:36:34 ==> oel-7: Clearing any previously set network interfaces...
11:36:34 ==> oel-7: Preparing network interfaces based on configuration...
11:36:34     oel-7: Adapter 1: nat
11:36:34 ==> oel-7: Forwarding ports...
11:36:34     oel-7: 22 (guest) => 2222 (host) (adapter 1)
11:36:34 ==> oel-7: Running 'pre-boot' VM customizations...
11:36:34 ==> oel-7: Booting VM...
11:36:35 ==> oel-7: Waiting for machine to boot. This may take a few minutes...
11:36:35     oel-7: SSH address: 127.0.0.1:2222
11:36:35     oel-7: SSH username: vagrant
11:36:35     oel-7: SSH auth method: private key
11:41:36 Timed out while waiting for the machine to boot. This means that
11:41:36 Vagrant was unable to communicate with the guest machine within
11:41:36 the configured ("config.vm.boot_timeout" value) time period.
11:41:36 
11:41:36 If you look above, you should be able to see the error(s) that
11:41:36 Vagrant had when attempting to connect to the machine. These errors
11:41:36 are usually good hints as to what may be wrong.
11:41:36 
11:41:36 If you're using a custom box, make sure that networking is properly
11:41:36 working and you're able to connect to the machine. It is a common
11:41:36 problem that networking isn't setup properly in these boxes.
11:41:36 Verify that authentication configurations are also setup properly,
11:41:36 as well.
11:41:36 
11:41:36 If the box appears to be booting properly, you may want to increase
11:41:36 the timeout ("config.vm.boot_timeout") value.
11:41:36 :x-pack-elasticsearch:qa:vagrant:vagrantOel7#up FAILED
11:41:36 :x-pack-elasticsearch:qa:vagrant:vagrantOel7#up (Thread[Task worker for ':' Thread 6,5,main]) completed. Took 5 mins 35.063 secs.
```

Another one was https://internal-ci.elastic.co/job/elastic+x-pack-elasticsearch+5.6+packaging-tests/234/console

**Comment by cbuescher** [comment]
Pinged infra channel to see if this is related to https://github.com/elastic/infra/commit/d09b55b4efed19f699dda7e761fbbc4af58110ed since the failure seems to have started around the same day (Nov 22) https://internal-ci.elastic.co/job/elastic+x-pack-elasticsearch+5.6+packaging-tests/

**Comment by cbuescher** [comment]
Opened https://github.com/elastic/infra/issues/8097 over at Infra.

**Comment by danielmitterdorfer** [comment]
Vagrant 2.2.2, which fixes that issue, has now been deployed on our CI workers.

**Comment by talevy** [comment]
I re-opened this issue because it has popped up again with `master`

CI link: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+packaging-tests/448/console

logs:

```
:x-pack:qa:vagrant:vagrantOel7#up (Thread[Execution worker for ':' Thread 6,5,main]) started.

> Task :x-pack:qa:vagrant:vagrantOel7#up
Task ':x-pack:qa:vagrant:vagrantOel7#up' is not up-to-date because:
  Task has not declared any outputs despite executing actions.
Custom actions are attached to task ':x-pack:qa:vagrant:vagrantOel7#up'.
Starting process 'command 'vagrant''. Working directory: /var/lib/jenkins/workspace/elastic+elasticsearch+master+packaging-tests/x-pack/qa/vagrant Command: vagrant up oel-7 --provision --provider virtualbox
Successfully started process 'command 'vagrant''
Bringing machine 'oel-7' up with 'virtualbox' provider...
==> oel-7: Importing base box 'elastic/oraclelinux-7-x86_64'...

[KProgress: 10%
[KProgress: 20%
[KProgress: 30%
[KProgress: 50%
[KProgress: 60%
[KProgress: 70%
[KProgress: 80%
[KProgress: 90%
[K==> oel-7: Matching MAC address for NAT networking...
==> oel-7: Checking if box 'elastic/oraclelinux-7-x86_64' is up to date...
==> oel-7: Setting the name of the VM: elasticelasticsearchmasterpackaging-tests_oel-7_1550103023782_56237
==> oel-7: Fixed port collision for 22 => 2222. Now on port 2200.
==> oel-7: Clearing any previously set network interfaces...
==> oel-7: Preparing network interfaces based on configuration...
    oel-7: Adapter 1: nat
==> oel-7: Forwarding ports...
    oel-7: 22 (guest) => 2200 (host) (adapter 1)
==> oel-7: Running 'pre-boot' VM customizations...
==> oel-7: Booting VM...
==> oel-7: Waiting for machine to boot. This may take a few minutes...
    oel-7: SSH address: 127.0.0.1:2200
    oel-7: SSH username: vagrant
    oel-7: SSH auth method: private key
Timed out while waiting for the machine to boot. This means that
Vagrant was unable to communicate with the guest machine within
the configured ("config.vm.boot_timeout" value) time period.

If you look above, you should be able to see the error(s) that
Vagrant had when attempting to connect to the machine. These errors
are usually good hints as to what may be wrong.

If you're using a custom box, make sure that networking is properly
working and you're able to connect to the machine. It is a common
problem that networking isn't setup properly in these boxes.
Verify that authentication configurations are also setup properly,
as well.

If the box appears to be booting properly, you may want to increase
the timeout ("config.vm.boot_timeout") value.


> Task :x-pack:qa:vagrant:vagrantOel7#up FAILED
```

**Comment by rjernst** [comment]
Now that 7.0 is released, 5.6 is no longer in CI, so this can be closed.
